src/wasm-lib/kcl/src/settings/types/mod.rs

```diff
@@ -280,6 +284,57 @@ pub struct ModelingSettings {
     pub show_scale_grid: bool,
 }
 
+fn named_view_point_version_one() -> f64 {
+    return 1.0;
+}
```

Comment:
Can we change this to "named_view_version" which is more generic and has the intent that the version can change?